### DISCUSSION TITLE
Systemd Service file

### DISCUSSION THREAD

**Discussion by tandy-1000** [slskd/slskd#82] [Show and tell]
I made a Systemd service file, it might be useful to include some variation of this in the git repo for users who don't want to use docker.

It checks that the network is online, and the share path exists.
```
[Unit]
Description=slskd
After=network-online.target
Wants=network-online.target
ConditionPathExists=/music

[Service]
User=slskd
WorkingDirectory=/opt/slskd
ExecStart=/opt/slskd/slskd
Restart=always

[Install]
WantedBy=multi-user.target
```

**Comment by jpdillingham** [comment]
Thanks!  I'll be sure to include this with the next documentation update

**Comment by BachoSeven** [comment]
I recently found out about the project and I'd like to make a package for the Arch User Repository, so this service file would certainly be included. Perhaps the shares path could also be made into an environmental variable like SLSKD_SLSK_USERNAME etc., so one wouldn't have to edit the service file itself(?)

On this note, is there any plans to package this program for any other distribution, and/or could you possibly provide guidelines for building the binaries from source? I could package the binaries you release, but for the AUR it would make more sense to build it. Would running https://github.com/slskd/slskd/blob/master/bin/publish#L97 be enough to build it?

**Reply by jpdillingham** [reply]
Another user created issue #427; if you're still interested could you elaborate on what an AUR is in that issue? I haven't looked at it personally but I'm interested to get this going.

**Comment by jpdillingham** [comment]
I dropped the ball on this!  I've created issue #542 so I won't forget again.
